src/Migrations/Schema/Column.php

```diff
@@ -4,7 +4,7 @@
 
 class Column
 {
-    const RESERVED_COLUMN_TYPE = '__reserved_column_type';
+    private const RESERVED_COLUMN_TYPE = '__reserved_column_type';
```

Comment:
__[PHPMD]__ The class Column has 21 fields. Consider redesigning Column to keep the number of fields under 15. ([see on Sider](https://sider.review/gh/repos/151422991/pulls/92#issue-4821595))

<details>

References:
- <https://phpmd.org/rules/codesize.html#toomanyfields>

</details>
<sub>posted by Sider</sub>